frontend/.eslintrc.cjs

```diff
@@ -0,0 +1,83 @@
+/**
+ * This is intended to be a basic starting point for linting in your app.
+ * It relies on recommended configs out of the box for simplicity, but you can
+ * and should modify this configuration to best suit your team's needs.
+ */
+
+/** @type {import('eslint').Linter.Config} */
+module.exports = {
+  root: true,
+  parserOptions: {
+    ecmaVersion: 'latest',
+    sourceType: 'module',
+    ecmaFeatures: {
+      jsx: true,
+    },
+  },
+  env: {
+    browser: true,
+    commonjs: true,
```

Comment:
We aren't using commonjs

Reply:
It give me the following error if I remove it or set it to `false`.

![image](https://github.com/DTS-STN/canadian-dental-care-plan/assets/114004123/0d01de4e-0f74-43ea-b18f-81b9495a75e5)

Reply:
So we have to enable it so eslint can lint eslint. 🤣

Reply:
I guess it's needed. I tried and failed. 🤷